docs/source/contributing/ci.adoc

```diff
@@ -60,3 +60,9 @@ link:https://ci.centos.org/job/minishift-release/[release]
 |link:https://travis-ci.org/minishift/minishift/branches[master], link:https://travis-ci.org/minishift/minishift/pull_requests[pull requests]
 |link:https://github.com/minishift/minishift/blob/master/.travis.yml[.travis.yml]
 |===
+
+[[nightly-job]]
+== Nightly Job
+
+A link:https://ci.centos.org/job/minishift-nightly[nightly job] has been configured to be run daily at midnight (2:20 AM UTC) which executes all tests against the `master` branch from the link:https://github.com/minishift/minishift[minishift] repo.
```

Comment:
@minishift/minishift-dev Please review this part of documentation.

Reply:
Nitpick: s/to be run/to run

Reply:
2:20 am UTC? Is there somewhere a timezone which with 20min offset? Why don't we run it midnight UTC?

Reply:
So why is this nightly in a new section instead of incorporating it into the table as part of the CentOS CI row?

Reply:
@budhrg Why 2:20 AM UTC? Why not 00:00AM UTC.

Reply:
> So why is this nightly in a new section instead of incorporating it into the table as part of the CentOS CI row?

@hferentschik Personally I feel a separate section makes sense as  nightly tests should not be considered as 100% continuous integration.

Reply:
@LalatenduMohanty @hferentschik , I had specified midnight in the job configuration here https://github.com/minishift/minishift-ci-jobs/blob/master/minishift-ci-index.yaml#L248. Not sure how it is starting after 2:20 hrs from midnight. See `System clock time` timestamps here https://ci.centos.org/job/minishift-nightly/10/consoleFull.

![screenshot from 2017-11-27 18-11-33](https://user-images.githubusercontent.com/1132451/33267178-83387e74-d39e-11e7-8f83-85ddb858d450.png)